csharp_dotnetcore/samples.ruleset

```diff
@@ -0,0 +1,86 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Microsoft Managed Recommended Rules" Description="These rules focus on the most critical problems in your code, including potential security holes, application crashes, and other important logic and design errors. It is recommended to include this rule set in any custom rule set you create for your projects." ToolsVersion="10.0">
+  <Localization ResourceAssembly="Microsoft.VisualStudio.CodeAnalysis.RuleSets.Strings.dll" ResourceBaseName="Microsoft.VisualStudio.CodeAnalysis.RuleSets.Strings.Localized">
+    <Name Resource="MinimumRecommendedRules_Name" />
+    <Description Resource="MinimumRecommendedRules_Description" />
+  </Localization>
+  <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
```

Comment:
The ManagedCodeAnalysis rules (this particular XML Element) are not needed and can all be deleted. We're not using that suite of tools, and are instead using the newer stack.

Reply:
All right, this is from Dave's PR 301 original ruleset. I'll copy the latest from here which seems to be correct and doesn't have the managed code rules: https://github.com/Microsoft/BotBuilder-Samples/pull/301/files